crates/sqlite_store/schema/schema_0.sql

```diff
@@ -0,0 +1,67 @@
+-- schema version control
+CREATE TABLE version (version INTEGER) STRICT;
+INSERT INTO version VALUES (1);
+
+-- network is the valid network for all other table data
+CREATE TABLE network
+(
+    name TEXT UNIQUE NOT NULL
+) STRICT;
+
+-- keychain is the json serialized keychain structure as JSONB,
+-- descriptor_id is a sha256::Hash id of the descriptor string w/o the checksum,
+-- descriptor is the complete descriptor string,
+-- last active index is a u32
+CREATE TABLE keychain
+(
+    keychain      BLOB PRIMARY KEY NOT NULL,
+    descriptor    TEXT UNIQUE      NOT NULL,
+    descriptor_id BLOB UNIQUE      NOT NULL,
+    last_revealed INTEGER
+) STRICT;
+
+-- hash is block hash hex string,
+-- block height is a u32,
+-- previous block hash hex string
+CREATE TABLE block
+(
+    hash   TEXT PRIMARY KEY NOT NULL,
+    height INTEGER          NOT NULL
+) STRICT;
+
+-- txid is transaction hash hex string (reversed)
+-- whole_tx is a consensus encoded transaction,
+-- last seen is a u64 unix epoch seconds
+CREATE TABLE tx
+(
+    txid      TEXT PRIMARY KEY NOT NULL,
+    whole_tx  BLOB             NOT NULL,
+    last_seen INTEGER
+) STRICT;
+
+-- Outpoint txid hash hex string (reversed)
+-- Outpoint vout
+-- TxOut value as SATs
+-- TxOut script consensus encoded
+CREATE TABLE txout
+(
+    txid   TEXT    NOT NULL,
+    vout   INTEGER NOT NULL,
+    value  INTEGER NOT NULL,
+    script BLOB    NOT NULL,
```

Comment:
Same as for descriptor_id I don't know how useful it would be for debugging to have a human readable version of the script. Is there some particular reason you had in mind for making it human readable?

Reply:
maybe at a glance you could see if a script was OP_RETURN for example, but it probably makes more sense to just load the changesets into an app that can decode everything for you.